### ISSUE TITLE
Compiled packages website is down

### ISSUE THREAD

**Issue by crzsotona** [maurerr/Entware-backports#4]
http://pkg.entware-backports.tk/packages/Packages.html is down thus rendering this repository useless for regular users

**Comment by ShadowEO** [comment]
Yup, it's been inaccessible for at least a month, makes updating the entware installation on my mini-router with an unsupported kernel version for entware-ng impossible.

**Comment by crzsotona** [comment]
I've found second mirror on maurerr's snb forum post:
http://maurerr.github.io/packages/

**Comment by ShadowEO** [comment]
Thanks, though I'm hoping that it's temporary and not that he's discontinued the repository, considering the last big update was months ago (the last commit from two months ago didn't appear to be anything substantial)
